tests/conftest.py

```diff
@@ -87,3 +90,110 @@ def large_test_session(blank_postgis_session):
     logging.getLogger('sqlalchemy.engine').setLevel(logging.INFO) # Let's not log all the db setup stuff...
 
     yield blank_postgis_session
+
+# To maintain database consistency, objects must be added (and flushed) in this order:
+#   Network
+#   Station, History
+#   Variable
+#   Observation
+#
+# This imposes an order on the definition of session fixtures, and on the nesting of describe blocks that use them.
+
+@fixture
+def network1():
+    return Network(id=1, name='Network 1')
+
+@fixture
+def network2():
+    return Network(id=2, name='Network 2')
+
+@fixture
+def station1(network1):
+    return Station(id=1, network_id=network1.id)
```

Comment:
FWIW, you *should* also be able to say:

```Station(network=network1)```

That's the more ORMy way to do it.